sdk/formrecognizer/azure-ai-formrecognizer/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,630 @@
+    print("Document has type {}".format(document.doc_type))
+    print("Document has document type confidence {}".format(document.confidence))
+    print("Document was analyzed with model with ID {}".format(result.model_id))
+    for name, field in document.fields.items():
+        field_value = field.value if field.value else field.content
+        print("......found field of type '{}' with value '{}' and with confidence {}".format(field.value_type, field_value, field.confidence))
+
+
+# iterate over tables, lines, and selection marks on each page
+for page in result.pages:
+    print("\nLines found on page {}".format(page.page_number))
+    for line in page.lines:
+        print("...Line '{}'".format(line.content))
+    for word in page.words:
+        print(
+            "...Word '{}' has a confidence of {}".format(
+                word.content, word.confidence
+            )
+        )
+    if page.selection_marks:
+        print("\nSelection marks found on page {}".format(page.page_number))
+        for selection_mark in page.selection_marks:
+            print(
+                "...Selection mark is '{}' and has a confidence of {}".format(
+                    selection_mark.state, selection_mark.confidence
+                )
+            )
+
+for i, table in enumerate(result.tables):
+    print("\nTable {} can be found on page:".format(i + 1))
+    for region in table.bounding_regions:
+        print("...{}".format(i + 1, region.page_number))
+    for cell in table.cells:
+        print(
+            "...Cell[{}][{}] has text '{}'".format(
+                cell.row_index, cell.column_index, cell.content
+            )
+        )
+print("-----------------------------------")
+```
+
+### Training a custom model
+
+Differences between the versions:
+- Files for building a new model for version `3.2.x` can be created using the labeling tool found [here](fr-labeling-tool).
+- In version `3.1.x` the `use_training_labels` kwarg was used to indicate whether to use labeled data was when creating the custom model.
+- In version `3.2.x` the `use_training_labels` kwargs is not supported since training must be carried out with labeled training documents. In order to extract key-value pairs from a document, please refer to the prebuilt model "prebuilt-document" which extracts entities, key-value pairs, and layout from a document. 
+- When using the latest service API version `2021-09-30-preview` models no longer include submodels, instead a model can analyze different document types.
+- In version `3.1.x`, when training without labels the service would returna model that recognized key-value pairs in the training data. When using the library with version `3.2.x`, users can instead use the prebuilt model `prebuilt-document` to get this information from documents without the need to train a new model. `prebuilt-document` returns key-value pairs, entities, text layout, among other useful information about the documents passed in for analysis.
```

Comment:
this feels like a repetition of the last 2 points. Consider removing it

Reply:
I don't think we should say 3.1.x returned key-value pairs, these were exposed as RecognizedForm.fields.

Reply:
So I actually removed this part in favor of the previous explanation like Mari suggested since I do mention using "prebuilt-document" in one of the previous points.